src/page/home/interact/interact.tsx

```diff
@@ -0,0 +1,75 @@
+import type { FC } from 'react'
+import { useMemo } from 'react'
+import { useTranslation } from 'react-i18next'
+import type { CardBackground } from '@/component/card/card'
+import { Card } from '@/component/card/card'
+import classNames from 'classnames'
+import './interact.scss'
+import '../i18n/index'
+
+type InteractCard = {
+  category: string
+  title: string
+  _tag: string
```

Comment:
Can be named `tag` instead of `_tag`, the underscore prefix referring to a meta-information, which is not the case here, as far as I understand.

```suggestion
  tag: string
```